### ISSUE TITLE
Private Mode "Learn More" new copy/design

### ISSUE THREAD

**Issue by kylehickinson** [brave/brave-ios#280]
The no-tabs private mode state that displays the usual information about private mode currently has a "Learn more about private tabs." button to have parity with 1.6.

This will be need to be replaced by longer copy explaining more details around private mode, or by instead showing that longer copy some different way upon clicking the "Learn more" button.

![private-notabs](https://images.zenhubusercontent.com/5b4cfb87b69de9433c0523d6/34014459-449f-44a3-9391-911363d3da3b)

**Comment by tmancey** [comment]
@anthonypkeane @jhreis has a decision been made as to the copy and how it will be displayed please? Please also see #220

**Comment by anthonypkeane** [comment]
@tomlowenthal can we sync on this and finalize the copy needed here.

Once this is final, @jamesmudgett you and I can finalize the design.

**Comment by tildelowengrimm** [comment]
Top screen is:

> Private Tabs aren't saved in Brave, but they don't make you anonymous online. Sites you visit in a private tab won't show up in your history and their cookies always vanish when you close them — there won't be any trace of them left in Brave. Your mobile carrier (or the owner of the WiFi network or VPN you're connected to) can see which sites you visit and and those sites will learn your public IP address, even in Private Tabs.


Below the break:

> Using Private Tabs only changes what Brave does on your device, it doesn't change anyone else's behavior.
>
> Sites always learn your IP address when you visit them. From this, they can often guess roughly where you are — typically your city. Sometimes that location guess can be much more specific. Sites also know everything you specifically tell them, such as search terms. If you log into a site, they'll know you're the owner of that account. You'll still be logged out when you close the Private Tabs because Brave will throw away the cookie which keeps you logged in.
>
> Whoever connects you to the Internet (your ISP) can see all of your network activity. Often, this is your mobile carrier. If you're connected to a WiFi network, this is the owner of that network, and if you're using a VPN, then it's whoever runs that VPN. Your ISP can see which sites you visit as you visit them. If those sites use HTTPS, they can't make much more than an educated guess about what you do on those sites. But if a site only uses HTTP then your ISP can see everything: your search terms, which pages you read, and which links you follow.
>
> If an employer manages your device, they might also keep track of what you do with it. Using Private Tabs probably won't stop them from knowing which sites you've visited. Someone else with access to your device could also have installed software which monitors your activity, and Private Tabs won't protect you from this either.

**Comment by srirambv** [comment]
That below section is too much text on small screens. Having just the top section text is sufficient in my opinion. 

cc: @kjozwiak @LaurenWags

**Comment by LaurenWags** [comment]
+1 to https://github.com/brave/brave-ios/issues/280#issuecomment-428395576 by @srirambv 

I think there was discussion before about having the 'below the break text' above in the 'Learn more about private tabs.' link - is that still an option? that seems like it would work best for the smaller screens.

**Comment by jamesmudgett** [comment]
![brave-pd.png](https://images.zenhubusercontent.com/599b64968f62dc7798c3931e/d8ce4410-8f6e-4679-aa79-0692e1ad3401)

Learn More tap should open expanded (scrollable) text.

**Comment by kjozwiak** [comment]
Agree with @srirambv & @LaurenWags, I don't think displaying a huge wall of text to users when they tap on "Learn more" is the best approach. Even though it's not displayed as the default and requires user interaction, it's white text, on a dark/black background that tries to explain something technical. I don't think explaining `Private Browsing` in a paragraphed format is the right approach.

Now that @Brave-Matt & @tomlowenthal are working on a knowledge base, I think a better approach would be:

* Display a small paragraph that basically explains the absolute minimal regarding PB
* Taping on `Learn more` would take the user to a KB article which will have a better explanation, with screenshots (a lot better then a wall of unformatted text, meaning bullets, screens etc..).

**Comment by tildelowengrimm** [comment]
Paragraph text should be left-aligned, not centered.

**Comment by kylehickinson** [comment]
Seems like there's still some discussion around the final decision. Re-adding the blocked label

**Comment by tildelowengrimm** [comment]
tl;dr: Long text in the product is good actually.

As we learned from our intern who researched it this summer, people who click "learn more" want more information. Often people look for longer text even when we don't offer a "learn more" affordance — they just don't find it. Longer text explaining things is *specifically* what they just requested with their click.

I've said it before and I'll say it again. Requiring a network interaction to view an outside page in order to understand what a privacy feature does is a dark pattern. The text explaining this confusing feature which someone has literally just asked to understand should be available immediately, and in-context. They should need to wait for a load, or have a context switch, or potentially reveal that they just tried to use a privacy feature.

Honestly, I'm tired of having to re-litigate this over and over again.

**Comment by kylehickinson** [comment]
@tomlowenthal So is the final design basically https://github.com/brave/brave-ios/issues/280#issuecomment-428658577 but with left-aligned text?

**Comment by tildelowengrimm** [comment]
Yep

**Comment by kylehickinson** [comment]
OK awesome

@jamesmudgett You good with that? Do you want to update your design to show left-aligned text?

**Comment by anthonypkeane** [comment]
@jamesmudgett really doesn't need to left alight the text here in the image. 

It is James's design, but left aligned @kylehickinson  - hopefully that is clear.

**Comment by kylehickinson** [comment]
Yeah its definitely clear, and he doesn't have to update it. Just wanted to make sure he's fine with the change itself.

**Comment by jamesmudgett** [comment]
It seems weird to left align content when everything else is centered. What's the concern with center alignment?

**Comment by tildelowengrimm** [comment]
Center-aligned paragraphs are difficult to read, especially for folks who have difficulty visually-parsing lines to begin with.
